index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
```

Comment:
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/75784269/205629125-23e4dca8-c21f-4e63-93d9-083f79450287.png">

The image is not properly aligned.